flytekit/deck/renderer.py

```diff
@@ -0,0 +1,22 @@
+import typing
+from abc import ABC
+
+import pandas as pd
+
+
+class Renderer(ABC):
```

Comment:
Let's rename
```suggestion
class Renderable(Protocol):
```

Also I think Renderable should be a protocol